pkgs/by-name/me/mealie/package.nix

```diff
@@ -0,0 +1,103 @@
+{ lib, callPackage, fetchFromGitHub, gnused, nixosTests, python3, python3Packages, writeShellScript }: let
+  version = "1.0.0-RC2";
+  src = fetchFromGitHub {
+    owner = "mealie-recipes";
+    repo = "mealie";
+    rev = "v${version}";
+    sha256 = "sha256-/sht8s0Nap6TdYxAqamKj/HGGV21/8eYCuYzpWXRJCE=";
+  };
+
+  frontend = callPackage (import ./mealie-frontend.nix src version) { };
+
+  # TODO  Add crfpp in runtime dependencies for ingredient quantities NLP
+
+in python3Packages.buildPythonPackage rec {
+  pname = "mealie";
+  inherit version src;
+  format = "pyproject";
+
+  patches = [
+    ./mealie_statedir.patch
+    ./mealie_init_db.patch
+    ./mealie_logger.patch
+  ];
+
+  nativeBuildInputs = [
+    python3Packages.poetry-core
+    python3.pkgs.pythonRelaxDepsHook
```

Comment:
You do not want to mix python3.pkgs and python3Packages as they could point to different pythons if overwrites are done incorrect, resulting in strange bugs and mismatches. Please choose either.